pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -191,57 +198,17 @@ func (configgen *ConfigGeneratorImpl) buildSidecarListeners(env *model.Environme
 	noneMode := node.GetInterceptionMode() == model.InterceptionNone
 	listeners := make([]*xdsapi.Listener, 0)
 
-	actualWildcard, actualLocalHostAddress := getActualWildcardAndLocalHost(node)
+	_, actualLocalHostAddress := getActualWildcardAndLocalHost(node)
 
 	if mesh.ProxyListenPort > 0 {
-		inbound := configgen.buildSidecarInboundListeners(env, node, push, proxyInstances)
-		outbound := configgen.buildSidecarOutboundListeners(env, node, push, proxyInstances)
```

Comment:
I don't undestand this change - please add comments ( if it is really related with the feature ). 
I want to be sure the behavior when the option is not set is identical with what we had before.

Reply:
Fixed: use the old when inbound listener doesn't capture all ports